draft-irtf-cfrg-hpke.md

```diff
@@ -240,27 +240,60 @@ informative:
 This document describes a scheme for hybrid public-key encryption (HPKE).
 This scheme provides a variant of public-key encryption of arbitrary-sized
 plaintexts for a recipient public key. It also includes three authenticated
-variants, including one which authenticates possession of a pre-shared key,
-and two optional ones which authenticate possession of a KEM private key.
-HPKE works for any combination of an asymmetric key encapsulation mechanism
-(KEM), key derivation function (KDF), and authenticated encryption with
+variants, including one that authenticates possession of a pre-shared key
+and two optional ones that authenticate possession of a key encapsulation
+mechanism (KEM) private key. HPKE works for any combination of an asymmetric
+KEM, key derivation function (KDF), and authenticated encryption with
 additional data (AEAD) encryption function. Some authenticated variants may not
 be supported by all KEMs. We provide instantiations of the scheme using widely
-used and efficient primitives, such as Elliptic Curve Diffie-Hellman key
-agreement, HKDF, and SHA2.
+used and efficient primitives, such as Elliptic Curve Diffie-Hellman (ECDH) key
+agreement, HMAC-based key derivation function (HKDF), and SHA2.
 
 This document is a product of the Crypto Forum Research Group (CFRG) in the IRTF.
 
 --- middle
 
+{::comment}
+[rfced] Please insert any keywords (beyond those that appear in the
+title) for use on https://www.rfc-editor.org/search.
+{:/comment}
+
+{::comment}
+[rfced] Please ensure that the guidelines listed in Section 2.1 of
+RFC 5743 (https://www.rfc-editor.org/rfc/rfc5743#section-2.1) have
+been adhered to in this document.
+{:/comment}
+
+{::comment}
+[rfced] "public key" vs. "public-key"
+in the attributive position (before a noun).
+We see both forms are used in this document; do you have a preference?
+
+For example, in the original:
+  Hybrid Public Key Encryption (the title; no hyphen)
+  and a couple instances of
+  public key encryption / validation
+
+vs. 10 instances in the document of
+  public-key encryption / scheme / operations / validation
+
+In general, we have received advice that it should not be
+hyphenated in the attributive position, and it is listed here:
+https://www.rfc-editor.org/materials/terms-online.txt
+
+We note that the IANA registry does not contain the hyphen
+(https://www.iana.org/assignments/hpke/), so depending on your
+decision here, there may be an update to its title.
```

Comment:
Personally, I generally prefer "public-key" in this position, but don't feel strongly.  And the terms document seems pretty clear.  So I have normalized on "public key" throughout.